cdk/package.json

```
@@ -7,6 +7,7 @@
     "test": "jest",
     "format": "prettier --write \"{lib,bin}/**/*.ts\"",
     "lint": "eslint lib/** bin/** --ext .ts --no-error-on-unmatched-pattern",
+    "lintfix": "eslint lib/** bin/** --ext .ts --no-error-on-unmatched-pattern --fix",
```

minor: not a huge fan of this. I'd always prefer to have the IDE format files on save to avoid annoying "fix linter" commits. Alternatively, `npm run lint -- --fix` should suffice here too.

---

Some developers hate having their IDE set up that way, so allowing this is beneficial. I would opt for the following syntax, however:

```suggestion
    "lint:fix": "npm run lint -- --fix",
```